platform/qt/src/qmapboxgl.cpp

```diff
@@ -618,11 +841,20 @@ void QMapboxGL::resize(const QSize& size, const QSize& framebufferSize)
         { static_cast<uint32_t>(size.width()), static_cast<uint32_t>(size.height()) });
 }
 
-void QMapboxGL::addAnnotationIcon(const QString &name, const QImage &sprite)
+/*!
+    Adds an \a icon to the annotation icon pool. This can be later used by the annotation
+    functions to shown some drawing on the map by referencing the \a name.
+
+    This is different from addIcon() and it is part of the Annotation API. Annotations will
```

Comment:
Unlike using `addIcon()` for runtime styling, annotations added with `addPointAnnotation()` will survive style changes.